lib/gui/app/components/safe-webview.js

```diff
@@ -191,12 +191,25 @@ class SafeWebview extends react.PureComponent {
    * @param {Event} event - Event object
    */
   didGetResponseDetails (event) {
-    const HTTP_OK = 200
-    const HTTP_ERR = 400
+    // This seems to pick up all requests related to the webview,
+    // only care about this event if it's a request for the main frame
+    if (event.resourceType === 'mainFrame') {
+      const HTTP_OK = 200
+      const HTTP_ERR = 400
+
+      const pickedHeaders = _.pick(event.headers, [
+        'status'
+      ])
+      const pickedEvent = _.pick(_.assign({}, event, pickedHeaders), [
+        'httpResponseCode',
+        'requestMethod'
```

Comment:
If I'm not mistaken, this will exclude the previously picked "status" header (which would make it redundant)? Also, what does the status header contain?